content.js

```diff
@@ -20,7 +24,21 @@ function toggleFiles() {
 
 	for (const el of rows) {
 		if (el.querySelector('.content a[title^="."')) {
```

Comment:
It is possible to not hide dotfiles as things currently stand, since right now everything *does* go through the RegExp, but this line is still necessary to pull the anchor text (the filename text) out of the DOM so the name can be compared to the regex:

```
el.querySelector('.content a[title^="."')
```